etna/packages/etna-py/src/mountetna/metis.py

```diff
@@ -436,7 +445,8 @@ def start_upload(self, request: "UploadStartRequest") -> "UploadResponse":
         return response_obj
 
     def upload_blob(self, request: "UploadBlobRequest"):
-        response = self.session.post(
+        response = self._retry_request(
+            self.session.post,
```

Comment:
This isn't retryable -- a failure does not mean that the previous blob was not saved.  It's possible, depending on the error, that the previous upload DID succeed on saving to the file system but FAILED due to say the connection or some cleanup tasks.  Which is why the previous logic attempted to infer if restarting the upload was necessary.